spec/models/category_spec.rb

```diff
@@ -61,6 +61,32 @@
       end
     end
 
+    context 'for pileid-source Category' do
+      let(:category) { create(:category, name: 28301, source: 'pileid') }
```

Comment:
Describe this example — what all is in this Page Pile? — and include a link to the pile in a comment.

Probably the name should be a string, since that's how it gets saved. It works the same either way, but that will help other developers recognize that the `Category` records have string values and not integers for `name`.

Reply:
This comment is still relevant. A comment should describe PagePile 28301, so we have some idea of why 'America' is an expected result. The comment should also link to that pile, so we can see what else is in the pile and more easily make adjustments to the test later if we need to.